.github/workflows/audit-chromium-schedule-dates.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Restore previous run data
-        uses: dawidd6/action-download-artifact@71072fbb1229e1317f1a8de6b04206afb461bd67 # v3.1.2
+        uses: dawidd6/action-download-artifact@20319c5641d495c8a52e688b7dc5fada6c3a9fbc # v8
```

Comment:
Review note: I was wondering about the name "v8" here since it doesn't follow the `v{major}.{minor}.{patch}` format this line previously had; but no, `v8` is the name used upstream:

https://github.com/dawidd6/action-download-artifact/commit/20319c5641d495c8a52e688b7dc5fada6c3a9fbc